docs/messaging/gcp-pubsub.md

```diff
@@ -16,7 +16,49 @@ For Google Cloud Pub/Sub, the following additional attributes are defined:
 <!-- semconv messaging.gcp_pubsub(full,tag=tech-specific-gcp-pubsub) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| [`messaging.gcp_pubsub.message.link`](../attributes-registry/messaging.md) | int | An identifier for the link. This is *not* the message id. The number will be in between 0 and the max batch size. | `1` | Recommended |
```

Comment:
> An identifier for the link. This is *not* the message id. The number will be in between 0 and the max batch size.

With this description, it's not really clear to me what this attribute tells me. I'm also not a Pub/Sub user, so I wonder if this will be self-evident to Pub/Sub users?

Reply:
This is not documented/obvious. This is just additional telemetry data. Based on your inputs, I think it is more confusing than helpful. Removing the attribute.